blueprints/app/files/package.json

```diff
@@ -26,10 +26,20 @@
     "test:ember": "ember test"
   },
   "devDependencies": {
-    "@babel/core": "^7.25.2",
-    "<% if (!typescript) { %>@babel/eslint-parser": "^7.25.1",
-    "@babel/plugin-proposal-decorators": "^7.24.7",
-    "<% } %>@ember/optional-features": "^2.1.0",
+    "@babel/core": "^7.25.2<% if (!typescript) { %>",
+    "@babel/eslint-parser": "^7.25.1",
+    "@babel/plugin-proposal-decorators": "^7.24.7<% } %><% if (typescript) { %>",
+    "@ember-data-types/adapter": "5.4.0-alpha.108",
```

Comment:
Should we switch to the [real packages](https://github.com/emberjs/data/blob/main/guides/typescript/0-installation.md#using-canary) instead of the `@ember-data-types` ones?

It seems the blueprint already uses `~5.4.0-beta.11` which I assume also includes the types?